Makefile

```diff
@@ -2414,6 +2414,45 @@ datasets/sqwiki.autolabeled_revisions.20k_2016.json: \
 		--verbose > \
 	datasets/sqwiki.autolabeled_revisions.20k_2016.json
 
+datasets/sqwiki.human_labeled_revisions.5k_2016.json:
+	./utility fetch_labels \
+		https://labels.wmflabs.org/campaigns/sqwiki/57/ > \
+	datasets/sqwiki.human_labeled_revisions.5k_2016.json
+
+datasets/sqwiki.human_labeled_revisions.5k_2016.no_review.json: \
+		datasets/sqwiki.human_labeled_revisions.5k_2016.json
+	cat datasets/sqwiki.human_labeled_revisions.5k_2016.json | \
+	grep '"needs_review": false' > \
+	datasets/sqwiki.human_labeled_revisions.5k_2016.no_review.json
+
+datasets/sqwiki.autolabeled_revisions.20k_2016.no_review.json: \
+		datasets/sqwiki.autolabeled_revisions.20k_2016.json
+	cat datasets/sqwiki.autolabeled_revisions.20k_2016.json | \
+	grep '"needs_review": false' > \
+	datasets/sqwiki.autolabeled_revisions.20k_2016.no_review.json
+
+datasets/sqwiki.labeled_revisions.20k_2016.json: \
+		datasets/sqwiki.human_labeled_revisions.5k_2016.no_review.json \
+		datasets/sqwiki.autolabeled_revisions.20k_2016.no_review.json \
+		datasets/sqwiki.human_labeled_revisions.5k_2016.json
+	( \
+	  ./utility merge_labels \
+	    datasets/sqwiki.human_labeled_revisions.5k_2016.no_review.json \
+	    datasets/sqwiki.autolabeled_revisions.20k_2016.no_review.json; \
+	  cat datasets/sqwiki.human_labeled_revisions.5k_2016.json | \
+	  grep '"needs_review": true' | shuf -rn 4558 \
```

Comment:
Hi thanks for your suggestion! I myself had some doubts regarding  that construct. I suppose we should first do head, then shuf as we need as few 'needs review: true' as possible.